### DISCUSSION TITLE
The size of the data obtained by training with OpenSplat is half the size of the Python version

### DISCUSSION THREAD

**Discussion by Sqiuln** [pierotofy/OpenSplat#102] [General]
OpenSplat is really great! But I've encountered some strange phenomena when using it. When I trained on the official dataset for 3D Gaussian Splatting, I found that the size of the .ply files generated by the training is half the size of the Python version of 3DGS.This could potentially lead to differences in the rendering results between OpenSplat and 3D Gaussian Splatting.
Can you tell me what the reason for this phenomenon is? I set the number of training iterations to 30,000.
                    OpenSplat          3DgaussianSplatting
playroom        487,752kb         838,642kb
drjohnson       256,947kb         571,773kb
train                175,448kb         258,677kb
truck                282,510kb        631,085kb

**Comment by pierotofy** [comment]
I'm not sure! I don't generally use proprietary software.
